Makefile

```diff
@@ -469,18 +469,21 @@ E2E_REGISTRY_NAMESPACE     ?= eck-dev
 E2E_IMG_TAG                ?= $(IMG_VERSION)
 E2E_IMG                    ?= $(REGISTRY)/$(E2E_REGISTRY_NAMESPACE)/eck-e2e-tests:$(E2E_IMG_TAG)
 E2E_STACK_VERSION          ?= 8.6.1
-export TESTS_MATCH         ?= "^Test" #testing # can be overriden to eg. TESTS_MATCH=TestMutationMoreNodes to match a single test
+# regexp to filter tests to run
+export TESTS_MATCH         ?= "^Test"
 export E2E_JSON            ?= false
 TEST_TIMEOUT               ?= 30m
 E2E_SKIP_CLEANUP           ?= false
 E2E_DEPLOY_CHAOS_JOB       ?= false
-E2E_TAGS                   ?= e2e  # go build constraints potentially restricting the tests to run
-E2E_TEST_ENV_TAGS          ?= ""   # tags conveying information about the test environment to the test runner
+# go build constraints potentially restricting the tests to run
+E2E_TAGS                   ?= e2e
```

Comment:
I moved the comment otherwise all the spaces before the `#` are used for the variable value!